### PR TITLE
Remove kube imports in staging

### PR DESCRIPTION
**Pull request by ncdc**
Remove k8s.io/kubernetes imports from staging

### CODE REVIEW

**Review by fabianvf — APPROVED** [review]
lgtm

File: staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go [hunk 1]
```diff
@@ -867,30 +868,20 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensionsv1.CustomResour
...
-		if err != nil {
-			utilruntime.HandleError(err)
-			return nil, fmt.Errorf("the server could not properly serve the CR columns")
-		}
-		table, err := tableconvertor.New(columns)
-		if err != nil {
-			klog.V(2).Infof("The CRD for %v has an invalid printer specification, falling back to default printing: %v", kind, err)
+		var table rest.TableConvertor
+		if r.tableConverterProvider != nil {
+			table = r.tableConverterProvider.GetTableConverter(crd.Spec.Group, v.Name, crd.Status.AcceptedNames.Kind, crd.Status.AcceptedNames.ListKind)
```
Comment (fabianvf) [inline staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go]:
```
# k8s.io/apiextensions-apiserver/pkg/apiserver                                                                                                                                                                                                
../../kubernetes/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go:873:54: too many arguments in call to r.tableConverterProvider.GetTableConverter                                               
        have (string, string, string, string)                                                                                                                                                                                                 
        want (string, string, string)    
```

looks like the function takes `group, kind, listKind string`, you're passing `v.Name` as well

**Review by fabianvf — CHANGES_REQUESTED** [review]
nvm found one thing

File: staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go (hunk 1 above)
Comment (ncdc) [reply staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go]: Thanks, fixing

**Comment by ncdc** [comment]
Let's see how CI for https://github.com/kcp-dev/kcp/pull/1000 does

**Review by fabianvf — APPROVED** [review]
/lgtm and compiles

**Pull request merged**